javalib/src/main/scala/java/util/LinkedList.scala

```diff
@@ -0,0 +1,367 @@
+package java.util
+
+import scala.annotation.tailrec
+import scala.collection.JavaConversions._
+
+class LinkedList[E]() extends AbstractSequentialList[E]
+    with List[E] with Deque[E] with Cloneable with Serializable {
+
+  def this(c: Collection[_ <: E]) = {
+    this()
+    addAll(c)
+  }
+
+  import LinkedList._
+
+  private var head: Node[E] = null
+  private var last: Node[E] = null
+
+  private var _size = 0
+
+  def getFirst(): E =
+    if (_size <= 0)
+      throw new NoSuchElementException()
+    else peekFirst()
+
+  def getLast(): E =
+    if (_size <= 0)
+      throw new NoSuchElementException()
+    else peekLast()
```

Comment:
Ah I see you over-interpreted a bit my earlier comment about the throws. In these two cases (getFirst and getLast, and only those), they were OK the way they were, because they're in expression position here.

In fact, by writing them this way now, you break two other style rules (both documented):

1. Either both of then/else should be on the same line, or none of them (but not one and not the other)
2. A method body which is not a one-liner nor a two-liner if/else should have {}

I suggest reverting to the `throw` on the same line as `if`. Or, if you prefer, you can do this:
```scala
def getFirst(): E = {
  if (_size <= 0)
    throw new NoSuchElementException()
  else
    peekFirst()
}
```